src/components/Courses/components/CourseCard/CourseCard.jsx

```diff
@@ -58,11 +58,11 @@ export const CourseCard = ({course, handleShowCourse, authorsList}) => {
 					<b>Created: </b>
 					<span>date</span>
 				</p>
-				<div>
+				<div className={styles.buttonsContainer}>
```

Comment:
I cannot find import for styles in this file.

Reply:
added